File: frontend/Business/tg_business_bot/Models/Drink.py
from pydantic import BaseModel
from typing import Optional

from Models.MenuItem import MenuItem
from Models.Nutrients import Nutrients

class Drink(MenuItem):
    volume: float

    def __init__(self,
                 category_id: int,
                 price: float,
                 volume: float,
                 name: str,
                 description: str,
                 cooking_time_minutes: int,
                 image_path: Optional[str] = None,
                 nutrients: Optional[Nutrients] = None,
                 id: Optional[int] = None,
                 **kwargs):
        super().__init__(
            id=id,
            category_id=category_id,
            price=price,
            name=name,
            description=description,
            image_path=image_path,
            cooking_time_minutes=cooking_time_minutes,
            volume=volume,
            nutrients=nutrients,
            **kwargs
        )

File: Domain/Models/Drink.cs
using Domain.DataBaseAccess;

namespace Domain.Models;

public class Drink : MenuItem, ITableDataBase
{
    public double Volume { get; set; }
    
    public Drink(){}
    public Drink(long? id,
        long categoryId, 
        int cookingTimeMinutes,
        double price,
        double volume,
        string name, 
        string description,
        string? imagePath,
        Nutrients? nutrients) : base(id, categoryId, cookingTimeMinutes, price, name, description, imagePath, nutrients)
    {
        Volume = volume;
    }
}